infrastructure/cloudconnect/README.md

```diff
@@ -0,0 +1,205 @@
+> Use the CloudFormation template _aws-vpn.yaml_ to create a CustomerGateway and enable the VPN connections. Both static and bgp configurations require the VPNGatewayId and PeerIp. For Static Routes, one is required to set StaticRoutesOnly to true, and the StaticRoute to the remote network cidr block. If multiple GCP static routes are required, go ahead and edit the template, using Route0 as an example.
+
+    ## Static Route Example (10.10.0.0/24 is the GCP subnet)
+    $ aws cloudformation create-stack --stack-name gcp-vpn-static --template-body file://aws-vpn.yaml --parameters ParameterKey=StaticRoutesOnly,ParameterValue=true ParameterKey=VPNGatewayId,ParameterValue=vgw-919078f8 ParameterKey=PeerIp,ParameterValue=35.185.31.1 ParameterKey=StaticRoute,ParameterValue="10.10.0.0/24"
+
+    ## Upon CREATE_COMPLETE, get the VpnId
+    $ aws cloudformation describe-stacks --stack-name gcp-vpn-static --query 'Stacks[0].Outputs[?OutputKey==`VpnId`].OutputValue' --output text
+    vpn-914251f0
+
+    ## BGP Example
+    $ aws cloudformation create-stack --stack-name gcp-vpn-bgp --template-body file://aws-vpn.yaml --parameters ParameterKey=VPNGatewayId,ParameterValue=vgw-919078f8 ParameterKey=PeerIp,ParameterValue=104.196.135.13
+
+    ## Upon CREATE_COMPLETE, get the VpnId
+    $ aws cloudformation describe-stacks --stack-name gcp-vpn-bgp --query 'Stacks[0].Outputs[?OutputKey==`VpnId`].OutputValue' --output text
+    vpn-8e4251ef
+
+
+> Once the gcp-vpn* stacks are _COMPLETE_, pipe the CustomerGateway(XML) configuration into the `gcp-vpn-buildconf.py` command (Note: Requires python mod jinja2). This will output a yaml configuration file for the gcp-vpn.jinja DeploymentManager template. A Google network url, in the same project as the PeerIp created above is required. See `./gcp-vpn-buildconf.py -h` for more information. For static routes, both local and remote traffic selector options are required. Local being the GCP subnet cidr block(s), and remote being the AWS VPC subnet cidr block(s).
+
+    ## Static Route Example
+    $ aws ec2 describe-vpn-connections --filter Name=vpn-connection-id,Values=vpn-914251f0 --query VpnConnections[0].CustomerGatewayConfiguration --output text | ./gcp-vpn-buildconf.py --network https://www.googleapis.com/compute/v1/projects/shared-vpc-host/global/networks/vpc --local-traffic-selector 10.10.0.0/24 --remote-traffic-selector 10.0.0.0/24 >> config.static.yaml
+
+    ## BGP Example
+    $ aws ec2 describe-vpn-connections --filter Name=vpn-connection-id,Values=vpn-8e4251ef --query VpnConnections[0].CustomerGatewayConfiguration --output text | ./gcp-vpn-buildconf.py --network https://www.googleapis.com/compute/v1/projects/shared-vpc-host/global/networks/vpc >> config.bgp.yaml
+
+
+> Launch _gcp-vpn.jinja_ deployment
+
+**NOTE:** Run within the Shared VPC host project
+
+    ## Static Route Example
+    $ gcloud deployment-manager deployments create vpn-static-to-aws                 compute.v1.targetVpnGateway  COMPLETED  []
```

Comment:
$ gcloud deployment-manager deployments create vpn-static-to-aws --config config.static.yaml
    NAME                              TYPE                         STATE      ERRORS  INTENT
    vpn-static-to-aws                 compute.v1.targetVpnGateway  COMPLETED  []